types/angular-material/index.d.ts

```diff
@@ -462,6 +462,7 @@ declare module 'angular' {
             openFrom(from: string | Element | Event | { top: number, left: number }): IPanelAnimation;
             closeTo(to: string | Element | { top: number, left: number }): IPanelAnimation;
             withAnimation(cssClass: string | { open: string, close: string }): IPanelAnimation;
+            duration(duration: number): IPanelAnimation;
```

Comment:
Does it support object props as well?
https://github.com/angular/material/blob/master/src/components/panel/panel.js#L3270-L3282

Reply:
Apparently so and I completely missed it. I'll push an updated commit.